doc/sphinx-guides/source/installation/config.rst

```diff
@@ -249,6 +249,12 @@ A Dataverse installation can alternately store files in a Swift or S3-compatible
 
 A Dataverse installation may also be configured to reference some files (e.g. large and/or sensitive data) stored in a web-accessible trusted remote store.
 
+When using integration tools, dataverse installation can be configured to allow out of band upload by setting the ``dataverse.files.\<id\>.allow-out-of-band-upload`` JVM option to ``true``.
+Files can be then uploaded by an integration tool with ``datasets/{id}/add`` api call, or uploaded directly to the storage and registerd in a dataset afterwards using the ``datasets/{id}/addFiles`` api call.
+Notice that using S3-storage with ``dataverse.files.\<id\>.upload-redirect`` JVM option enabled supersedes the ``allow-out-of-band-upload`` and will enable direct upload even with ``allow-out-of-band-upload`` not set (or set to false).
+In other words, ``dataverse.files.\<id\>.allow-out-of-band-upload`` option opens the ``datasets/{id}/add`` and ``datasets/{id}/addFiles`` api endpoints without redirecting uploads in the UI.
+Enabling the ``upload-redirect`` option allows then direct upload automatically, without the need of enabling the ``allow-out-of-band-upload`` (setting it to ``false`` does not have any effect in that case).
+
```

Comment:
Some remarks:

- There are some typos here and the language might need some adaption to match the other sections.
- More important: the option introduced here is not added to the option overview table below.
- I would appreciate adding crosslinks to the mentioned Native API calls (although this means adding markers to these sections in `api/native-api.rst` first)
- I would appreciate a more general introduction of the idea - maybe it would help to paraphrase what those API calls do, or give an example workflow.
- The second half about which setting is taking precedence seems a bit redundant, maybe this can be shortened?
- May I suggest using an option name that is more similar to "upload-redirect", like "upload-out-of-band"?
- What exactly is an "integration tool"? Is this relating to our "external tools" feature (should be linked and named as such then) or to some non-first-party tools provided by instance admins or users?
- Is there a good example how to use this new feature? Should we add an example to the API guide?

Reply:
We did some changes to the documentation, and we think it is better now and ready for the review.